resources/views/themes/default1/layouts/master.blade.php

```diff
@@ -354,6 +354,8 @@
             <!-- Main content -->
             <section class="content">
                 <div class="container-fluid">
+                    @if(Route::current()->getName() !== 'plans.index');
```

Comment:
Why are you checking the type also here with !==?

Reply:
It will not execute if the URL is "plan. index". because  create blade has an alert message for the popup separately